templates/whatson/event_page.html

```diff
@@ -1,3 +1,168 @@
 {% extends 'base_page.html' %}
 
-{% load static i18n wagtailcore_tags wagtailimages_tags wagtailmetadata_tags %}
\ No newline at end of file
+{% load static i18n wagtailcore_tags wagtailimages_tags wagtailmetadata_tags %}
+
+{% block html_class %}tna-template--pink-accent{% endblock %}
+
+{# Breadcrumb is included inside the header area #}
+{% block breadcrumb %}
+{% endblock %}
+
+{% block content %}
+  {% comment %} Area for hero component {% endcomment %}
+
+  <div class="event-details-banner">
+    <div class="tna-container tna-container--align-center tna-container--space-between">
+      <div class="tna-column tna-column--width-3-4 tna-column--container">
+        <div class="event-meta event-meta--inline">
+          <ul class="event-meta__list event-meta__list">
+            {% if page.start_date %}
+              <li class="event-meta__item">
+                {% include "includes/icon.html" with name="calendar" classname="event-meta__icon" %}
+
+                <div class="event-meta__details">
+                  <p>{{ page.start_date|date:"l j F, H:i" }}</p>
+                </div>
+              </li>
+            {% endif %}
+            {% if page.venue_type %}
+              <li class="event-meta__item">
+                {% include "includes/icon.html" with name="location" classname="event-meta__icon" %}
+
+                <div class="event-meta__details">
+                  <p>{{ page.venue_type }}</p>
+
+                  {% if page.venue_address %}
+                    <address>
+                      {{ page.venue_address }}
```

Comment:
I think this is a rich text block - on my local build it is rendering like this currently:

![Screenshot 2023-10-24 at 08 20 49](https://github.com/nationalarchives/ds-wagtail/assets/771869/a5b375d6-003a-4bcf-89b7-e5ed1f7ec1e3)

Reply:
Great thanks :)